src/routes/safe/container/index.tsx

```diff
@@ -1,76 +1,87 @@
-import * as React from 'react'
-import { useState } from 'react'
+import { GenericModal } from '@gnosis.pm/safe-react-components'
```

Comment:
Should we import this after react libraries? Just to keep same order between components

Reply:
done